glew will fail pkgconf test pipeline

If the pkgconf test pipeline is added to glew it will fail in the following way:<br/><br/>

```
2024/11/12 23:10:58 INFO running step "test/pkgconf"
2024/11/12 23:10:58 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:10:58 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:10:58 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:10:58 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:10:58 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + basename /home/build/melange-out/glew-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + dev_pkg=glew-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + apk info -L glew-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + grep -q ^Name: usr/lib/pkgconfig/glew.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + basename usr/lib/pkgconfig/glew.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + lib_name=glew uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + echo usr/lib/pkgconfig/glew.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + grep -q '^usr/lib/pkgconfig/glew.pc$\|^usr/share/pkgconfig/glew.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + pkgconf --exists glew uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN + pkgconf --print-errors --exists glew uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN Package osmesa was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN Perhaps you should add the directory containing `osmesa.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 WARN Package 'osmesa', required by 'glu', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:10:58 INFO pod e0c2cac5865638ff52901d65f4b7f0f3fd83268bd0c25e6d4c9bf6590b449d65 terminated
2024/11/12 23:10:58 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:10:58 INFO   workspace dir: /home/user/tmp/melange-workspace-1807894751
2024/11/12 23:10:58 INFO   guest dir: /home/user/tmp/melange-guest-177652707
2024/11/12 23:10:58 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/glew] Error 1```

--- a/glew.yaml
+++ b/glew.yaml
@@ -1,7 +1,7 @@
 package:
   name: glew
   version: 2.2.0
-  epoch: 2
+  epoch: 3
   description: "A cross-platform C/C++ extension loading library"
   copyright:
     - license: GPL-2.0-or-later
@@ -54,10 +54,14 @@ subpackages:
     dependencies:
       runtime:
         - glew
+        - mesa-dev
     pipeline:
       - uses: split/dev
       - runs: |
           rm -rf "${{targets.subpkgdir}}"/usr/lib/cmake
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

